src/transformer/get_event_function_map.php

```diff
@@ -96,10 +99,8 @@ function get_event_function_map() {
         '\mod_url\event\course_module_viewed' => 'mod_url\course_module_viewed',
         '\mod_wiki\event\course_module_viewed' => 'all\course_module_viewed',
         '\mod_workshop\event\course_module_viewed' => 'all\course_module_viewed',
-        '\totara_program\event\program_assigned' => 'totara_program\program_assigned',
-        '\core\event\calendar_event_created' => 'core\calendar_event_created',
-        '\core\event\calendar_event_updated' => 'core\calendar_event_updated',
-        '\core\event\calendar_event_deleted' => 'core\calendar_event_deleted',
```

Comment:
uh oh, were these left in on the previous merge?

Reply:
They were added before that, but this commit moves them up
https://github.com/yetanalytics/moodle-logstore_xapi/blame/master/src/transformer/get_event_function_map.php